### PR TITLE
🐛 PRTL-1327 resetting facets should not reset current tab

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

File: src/packages/@ncigdc/components/Aggregations/FacetResetButton.js [hunk 1]
```diff
@@ -26,26 +28,31 @@ const StyledLink = styled(Link, {
...
+  };
   const inCurrent = fieldInCurrentFilters({
     currentFilters: currentFilters.content || [],
     field,
   });
   return (
     <StyledLink
       style={{ display: inCurrent ? 'inline' : 'none', ...style }}
-      query={inCurrent ? newQuery : {}}
+      query={newQuery}
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Aggregations/FacetResetButton.js]: shouldn't need to import router to do this - can't this be handled by the link?

Comment (cheapsteak) [inline src/packages/@ncigdc/components/Aggregations/FacetResetButton.js]: This doesn't seem to work (nothing happens when you click the reset button)

```js
const FacetResetButton = ({
  field,
  currentFilters,
  style,
  ...props
}) => {
  const newFilters = removeFilter(field, currentFilters);
  const newQuery = {
    offset: 0,
    filters: newFilters,
  };
  const inCurrent = fieldInCurrentFilters({
    currentFilters: currentFilters.content || [],
    field,
  });
  return (
    <StyledLink
      merge="replace"
      style={{ display: inCurrent ? 'inline' : 'none', ...style }}
      query={newQuery}
    >
      <ShadowedUndoIcon /><Hidden>reset</Hidden>
    </StyledLink>
  );
};
```

Comment (Jephuff) [inline src/packages/@ncigdc/components/Aggregations/FacetResetButton.js]: I think you either need to use `toggle` and pass a filter that has just the values you want to remove, or maybe add a new merge function like `replaceAll` that would just be `q => q`

Comment (cheapsteak) [inline src/packages/@ncigdc/components/Aggregations/FacetResetButton.js]: Can we just use this? Works and can understand what's going on without having to know the intricacies of how `StyledLink` works

Comment (Jephuff) [inline src/packages/@ncigdc/components/Aggregations/FacetResetButton.js]: I have no objections.

**Pull request merged**